src/picosvg/geometric_types.py

```diff
@@ -151,6 +151,17 @@ def _overlap(start1, end1, start2, end2):
             return Rect(x1, y1, x2 - x1, y2 - y1)
         return None
 
+    def x2(self) -> float:
```

Comment:
how about `x_max` and `y_max`?

we could even make these `@property` getters as addition is cheap; then you will use them like `self.x_max` instead of `self.x_max()` which looks more pythonic

Reply:
Done